src/components/Edit/Edit.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import useEdit from '../../hooks/useEdit';
+
+export default function Edit() {
+  const [title, name, info, contact, error, message, submitEdit] = useEdit();
+
+  return (
+    <div>
+      {message}
+      Edit Page
+      {error && (
+        <p>
+          {error} <span onClick={() => setError('')}>Error With Edit</span>
+        </p>
+      )}
+      <form>
+        <label>
+          Title:
+          <input
+            type="text"
+            value={title}
+            onChange={(e) => setTitle(e.target.value)}
```

Comment:
It appears that the edit form does populate the fields correctly, however, the fields are unable to be changed. Consider using a different state object that prepopulates the existing fields.